{"name": "计算除法中正数、负数相除情况、被除数为0", "status": "passed", "attachments": [{"name": "log", "source": "8480be8f-d604-442d-bb6b-8412743223bc-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "a", "value": "0"}, {"name": "b", "value": "1"}, {"name": "expect", "value": "0"}], "start": 1626702690143, "stop": 1626702690143, "uuid": "307fb4c5-6dba-4198-83b3-064d7beba941", "historyId": "885141774483e931bda326260621e136", "testCaseId": "540ae9490c5546cd9bf0d8f636e8d760", "fullName": "test_calcul_yml.Test_calculate#test_division", "labels": [{"name": "feature", "value": "计算器模块"}, {"name": "story", "value": "除法计算成功"}, {"name": "suite", "value": "test_calcul_yml"}, {"name": "subSuite", "value": "Test_calculate"}, {"name": "host", "value": "PC-20200730PITZ"}, {"name": "thread", "value": "12300-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calcul_yml"}]}